### DISCUSSION TITLE
Unreal's C++ project and C# project in one solution file

### DISCUSSION THREAD

**Discussion by davidsanda** [nxrighthere/UnrealCLR#198] [Help] [answered]
The current development scenario that you describe calls for a separate solution, where one develops the c# side, debugging C# by attaching to unreal project. It works just fine.

Do you know if it is possible to configure the build to include third party _projects_ instead of compiled libraries?
I think it would be much more convenient to have the managed part and the unreal part in one solution.
I can do it manually, but to have it automated would be nice.

**Comment by nxrighthere** [comment, marked as answer]
Unreal Engine solution file (C++) and .NET solution file (C#) are two different fruits, and they can't be combined.

That said, there are ways to trigger building from one of the solutions (by configuring IDE, for example).
